ideascube/serveradmin/management/commands/csv2pkg.py

```diff
@@ -13,8 +16,30 @@
 Optional columns are "lang", "preview", "kind", "tags".
 Attributes "path" and "preview" (if provided) must be paths
 (relative to the directory containing the CSV) to existing files.
+
+csv2pkg also generates a yaml to add to the catalog.
+Some attributes are automically calculated from the generated package.
+Other attributes may be provided as command arguments.
 """
 
+
+def generate_catalog_yaml_metadata(path, metadata, _type='zipped-medias'):
+    return {
+        'all' : {
+            metadata['package_id']: {
```

Comment:
Just a pedantic comment on this:

> These fields are displayed in their alphabetical order.

There is no ordering guarantee for Python dictionaries.

Therefore, you saw the fields displayed in their alphabetical order, but you can't assume they always will be.

Even if you ran it 100 times and it always gave you the alphabetical ordering, it's only what happened in your experience, but there is no guaranteed that it will always happen.

Consider the current ordering to be random, and potentially changing every time.

---

As for the possibility to fix a given order, it should be doable, yes: http://stackoverflow.com/a/31609484